### ISSUE TITLE
Still Mistake of Redstone . . . (Redstone Torch Invert Mechanism and somemore...)

### ISSUE THREAD

**Issue by intagaming** [ImagicalMine/ImagicalMine#558]
Let say, in short, the mistake.
Let you take a look. (ImagicalMine dev378)

http://postimg.org/image/993g3l1at/

Then if i try to invert it:

http://postimg.org/image/5g4jn37ef/

nothing happen.

It should be inverted in Singleplayer World. I swear.

UPDATE: When i tried to turn off Redstone Torch with Level, it did't turn off the Redstone Wires that links to that Redstone Torch. Look at this:
First:
http://postimg.org/image/mpehx96qj/

Then:
http://postimg.org/image/8duzscokb/

Then:
http://postimg.org/image/3o523qxjv/

UPDATE: Daylight Sensor not working . . .

http://postimg.org/image/u2cvz1c0n/
http://postimg.org/image/483399c0n/

**Comment by thebigsmileXD** [comment]
All known issues, listed in "Ultra fast restone"

**Comment by intagaming** [comment]
So . . . that means its been fixed ? But still not work after Ultra Fast Redstone . . .
Or it havent been fixed yet ?

**Comment by thebigsmileXD** [comment]
They are in progress.. @aodzip fixed like.. 9 things after it :P

**Comment by aodzip** [comment]
Finally somebody find the bug ... I think you can find these lol

**Comment by remotevase** [comment]
Can be toggled, but sometimes when toggling it goes a crazy (creates a clock going at 0.2 ticks XD)

**Comment by thebigsmileXD** [comment]
True, told @aodzip once. It seems to appear next to redstone consumer (for example noteblocks)

**Comment by keithkfng** [comment]
@ImagicalMine/developers somebody mind confirming this bug in 0.14.2? Reopen if needed.

**Comment by remotevase** [comment]
confirmed
